build.gradle.kts

```diff
@@ -17,9 +17,24 @@ java {
     withSourcesJar()
 }
 
+modularity {
+    mixedJavaRelease(8)
+}
+
+tasks.getByName<Jar>("sourcesJar") {
+    exclude("module-info.class")
+}
```

Comment:
Why would the class file be part of the sources jar in the first place? Could you please elaborate?

Reply:
> Why would this class file be part of the sources jar in the first place? Could you please elaborate on why you had to add this exclusion? (also if there is a good reason please add it as a comment in the code)

After setting the mixedJavaRelease, the module-info.class will also be packaged into the sourcesJar. I think this may be a bug with the moduleplugin.

Reply:
Ok thank you. It might be worth reporting this to the plugin authors then. Do you have a link to an issue?

Reply:
It looks like there is a similar problem with the Javadoc jar too, by the way

Reply:
I fixed that in my branch and opened this issue: https://github.com/java9-modularity/gradle-modules-plugin/issues/220